docs/hardware/devices/heltec/index.mdx

```diff
@@ -57,6 +57,10 @@ Not recommended because of design issues! Support is being phased out. Use V3 in
 
 ## V3/V3.1
 
+:::caution
+Be careful when interacting with the USB-C port. This device does not have ESD protection for CP2102.
```

Comment:
To make this easier to understand, could I suggest adding "... for the CP2102 USB interface chip."  or similar?

Reply:
Updated with "the CP2102 USB to UART bridge chip".